### DISCUSSION TITLE
AI and Accessibility

### DISCUSSION THREAD

**Discussion by shawna-slh** [w3c/wai-mission#25] [High-level]
Artificial intelligence (AI) can improve accessibility. And there are challenges and cautions around AI and accessibility. Some are explored in W3C WAI's [AI and Accessibility Research Symposium 2023](https://www.w3.org/WAI/research/ai2023/).

What is W3C WAI role in AI and accessibility?
What should we consider doing within our scope?
What are priorities?
What is out of scope and/or not a priority?

**Comment by GeorgeKerscher** [comment]
AI indeed can be a powerful tool for people with disabilities to consume information. However, there is a huge caution that people do not expect AI to do what they should be doing themselves for accessibility. The overlay approach comes to mind. Website developers must be making their content Born Accessible. A developer could use AI to help them, but the human must be the final decision maker. I think it is important that we deliver this message. The notion that AI would take the place of proper accessible coding must be discouraged.
